### PR TITLE
Show the header programatically when requested

### PR DESCRIPTION
**Pull request by fjmorant**
This PR exposes a new method in the library called "showHeader" when you call this method the header will be shown again as at the beginning , this is very useful for the following situations:

 - The app rotates and you want to show the header again.
 - There are situation where the user scrolls really slowly to the bottom of the screen and the header looks halfway through, with this you can detect if you reach the bottom of a scrollview and then show it again.

This PR has been created in response to this issue created a few days ago https://github.com/benevbright/react-navigation-collapsible/issues/161

Let me know if you need me to do anything else @benevbright 

Also it would be possible to do the hide (pretty simple) but for our requirements is enough with show, but if you need my help I can create another PR for the hide.

### CODE REVIEW

File: src/core.tsx [hunk 1]
```diff
@@ -94,6 +95,12 @@ const useCollapsibleHeader = (
     collapsibleCustomHeaderHeight: customHeaderHeight,
   } = route.params || {};
 
+  const showHeader = () => {
+    positionY.setValue(0);
+    // Forces to show the header again
+    setHeaderStyle({ ...headerStyle });
```
Comment (benevbright) [inline src/core.tsx:101]: The header doesn’t show up without this?

**Review by benevbright — APPROVED** [review]
Amazing. Thanks @fjmorant 🙌🙌

File: src/core.tsx (hunk 1 above)
Comment (fjmorant) [reply src/core.tsx:101]: Hey,

I think it was necessary to force to run what is inside of the useEffect function (recalculate, progress, opacity, traslateY and call to navigation.setOptions, I will double check tomorrow morning if still works without it, bye

**Comment by fjmorant** [comment]
Hi @benevbright ,

when more or less would I be able to have this new feature in the main branch and a release?

Thanks

**Comment by benevbright** [comment]
@fjmorant Hi, after your testing about the comment above, I will merge/publish asap. :)

**Comment by fjmorant** [comment]
> @fjmorant Hi, after your testing about the comment above, I will merge/publish asap. :)

Ok, will check it soon

File: src/core.tsx (hunk 1 above)
Comment (fjmorant) [reply src/core.tsx:101]: You were right it is not necessary, I will change, thx

**Comment by fjmorant** [comment]
> > @fjmorant Hi, after your testing about the comment above, I will merge/publish asap. :)
> 
> Ok, will check it soon

@benevbright Removed unnecessary code

**Comment by benevbright** [comment]
It's included in `v5.8.1` thanks again.

**Comment by fjmorant** [comment]
> It's included in `v5.8.1` thanks again.

Awesome, thx very much

**Pull request merged**